rules/opentitan.bzl

```diff
@@ -783,7 +783,7 @@ def opentitan_flash_binary(
         dev_targets.extend(opentitan_binary(
             name = devname,
             deps = [depname],
-            extract_sw_logs_db = device in ["sim_dv", "sim_verilator"],
+            extract_sw_logs_db = device == "sim_dv",
```

Comment:
Should probably change above line 669 in `opentitan_rom_binary` too?

Reply:
Good catch, thanks! Done.